helpers.js

```diff
@@ -46,7 +46,14 @@ async function listGithubRequest(params, settings, path, searchParams){
         per_page: parsers.number(params.per_page)
     });
     if (Object.keys(searchParams).length > 0) {
-        path += "?" + new URLSearchParams(searchParams);
+        path += "?";
+        path += Object.entries(searchParams).map(([ key, value ]) => {
+            // if param is the query do not encode the value, otherwise the github does not parse it correctly
+            if (key === "q") {
+                return `q=${value}`
+            }
+            return `${key}=${encodeURIComponent(value)}`
+        }).join('&')
```

Comment:
`q=${value}` where key = `q` is still `${key}=${value}`

```suggestion
            // if param is the query do not encode the value, otherwise the github does not parse it correctly
            path += Object.entries(searchParams).map(([ key, value ]) =>
               `${key}=${key === "q" ? value : encodeURIComponent(value)}`
            ).join('&');
```